peps/conf.py

```diff
@@ -30,17 +30,20 @@
     ".rst": "pep",
 }
 
-# List of patterns (relative to source dir) to ignore when looking for source files.
+# List of patterns (relative to source dir) to include when looking for source files.
 include_patterns = [
     # Required for Sphinx
     "contents.rst",
     # PEP files
     "pep-????.rst",
     # PEP ancillary files
     "pep-????/*.rst",
+    # PEPs API
+    "api/*.rst",
```

Comment:
This doesn't exist, did you forget to `git add`?

Reply:
Oops, it's been a long conference and sprints :) Now added!